samples/canopen/canopen_tpdo/arduino_receive/arduino_receive.ino

```diff
@@ -64,30 +59,5 @@ void setup() {
 }
 
 void loop() {
```

Comment:
Why does this function exist? Can you add some documentation explaining why we need it?

Reply:
Arduino scripts just always have a loop and a setup.

Reply:
Ok